webpack/envs/clientDevelopmentConfig.js

```diff
@@ -56,29 +52,13 @@ export const clientDevelopmentConfig = {
     ],
   },
   plugins: [
-    new CaseSensitivePathsPlugin(),
```

Comment:
This was recently added because it helped us track down  a very pesky case sensitive bug. Why remove?

Reply:
If all of the following removals are due to plugin compat, we should comment them out with a 
```
TODO: Reenable once webpack 5 is supported
```

Reply:
Would you explain the particular bug in more detail or provide a link to it? I am intimately familiar with Apples case-insensitive file system and the many issues that it creates in particular when renaming files in git. However git has options to address this already and wonder if we could instead leverage those.

Reply:
https://github.com/artsy/force/pull/6462/ was the issue that prompted this — I had downcased a folder name accidently against convention, but imports would work locally with alternate casing. I'd very much like to be warned about this, regardless of where it happens.

Reply:
(Er, that's not the issue but rather the fix)